test/cypress/integration/widgets/filters_spec.ts

```diff
@@ -550,16 +553,29 @@ describe('Filters widget', () => {
             });
         }
 
+        function getRulesWithoutNonDynamicDropdownFlags(
```

Comment:
"WithoutNonDynamic" is a double negation, so may be hard to understand.

What about something like:
```suggestion
        function getRulesWithStaticValueInputs(
```
?

Reply:
I think that the `getRulesWithStaticValueInputs` is a bit unclear, as it's hard do establish which input values are `static` 😁 
What do you think about `getRulesWithoutInputOptions`?

Reply:
Actually I did a mistake in my comment as in fact the function seems to do: `getRulesWithoutStaticValueInputs` or `getRulesWithDynamicValueInputs` 🙂 

Two points:

1. Function implementation

Looking at the function implementation it seems like it does not exclude rules with static values as it is just removing `useStaticDropdown` field, but what if that field is already set to `true`? 🤔 Shouldn't we filter out rules having `useStaticDropdown : true` instead of taking rules which does not have it defined? I suspect it'll be more clear if we do so. Probably then `FilterRuleValueInputOptions` interface won't be necessary.

2. Function name

> What do you think about `getRulesWithoutInputOptions`?

The implementation suggests that, but I think the goal of this function and the context where it is used suggests something different. I suspect that the main goal of this function is to get only filter rules which are using HTTP requests to fetch data for value inputs and filter out rules which are using statically defined valid values. Am I right? If so, then maybe `getRulesUsingExternalDataSources` or `getRulesUsingDynamicDropdowns` or `getRulesNotUsingStaticDropdowns` or whatever showing the actual purpose? 🙂

Reply:
Oh, so if that's the intention, then why just not extend L580:
`expect(requestRules[index]).to.deep.equal(_.omit(testRules[index], ['newKey', 'newValues']))`
to omit also `useStaticDropdown` and have:
`expect(requestRules[index]).to.deep.equal(_.omit(testRules[index], ['newKey', 'newValues', 'useStaticDropdown']))`
?

That way you won't need neither `getRulesWithoutNonDynamicDropdownFlags` function nor `FilterRuleValueInputOptions` interface.

Reply:
All clarified in online meeting, so I'm resolving this discussion 🙂

Reply:
Seems like a good idea 💪 
Added the necessary adjustments 🎉